draft-ietf-taps-interface.md

```diff
@@ -849,26 +849,22 @@ inconsistent with the Selection Properties of the Connection yields an error.
 
 The following Message Context Parameters are supported:
 
-\[TODO: De-Duplicate with Properties in {{transport-props}}, find consensus on which Section to put them]
-
-### Lifetime
-
-\[TODO: De-Duplicate with {{send-lifetime}}]
+### Lifetime {#msg-lifetime}
 
 Lifetime specifies how long a particular Message can wait to be sent to the
 remote endpoint before it is irrelevant and no longer needs to be
 (re-)transmitted. When a Message's Lifetime is infinite, it must be
 transmitted reliably. The type and units of Lifetime are
 implementation-specific.
 
-### Niceness
+### Niceness {#msg-niceness}
 
-\[TODO: De-Duplicate with {{send-niceness}}]
+This property is a numeric (non-negative) value that represents an unbounded
+hierarchy of priorities. It can specify the priority of a Message, relative to
+other Messages sent over the same Connection or Connection Group (see
+{{groups}}), or the priority of a Connection, relative to other Connections in
+the same Connection Group.
```

Comment:
Previously, we had merged per-Message and per-Connection Niceness, but then come to realize that we rather want them separate.

I think this sentence is an artifact of this. 
The latter part refers to per-Connection Niceness (within a Connection Group), as opposed to per-Message Niceness (within the same Connection). I think here we only want per-Message Niceness, so the per-Connection part should be taken out.

Reply:
ACK!  I'm changing it only to "over the same Connection", dropping the group because this can make it confusing: what if I give a message a lower Niceness than all others in the group, yet the Connection the Message is on has a higher Niceness than the others? Then what does this mean?  All of this gets clear if the context of per-Message Niceness is only a single Connection. I think this is what you probably identified and I didn't think of so far - great catch!